handsontable/src/core.js

```diff
@@ -50,13 +50,34 @@ let activeGuid = null;
  *
  * ## How to call a method
  *
+ * ::: only-for javascript
  * ```js
  * // First, let's contruct Handsontable
  * const hot = new Handsontable(document.getElementById('example'), options);
  *
  * // Then, let's use the setDataAtCell method
  * hot.setDataAtCell(0, 0, 'new value');
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * // First, let's contruct Handsontable
+ * <HotTable
+ *   ref={hotRef}
+ *   settings={options}
+ * />
+ *
+ * ...
+ *
+ * // Then, let's use the setDataAtCell method
+ * hotRef.setDataAtCell(0, 0, 'new value');
```

Comment:
This is not correct - the API methods are not meant to be called on the ref, but on the Handsontable instance stored in the referenced component, so 

```
const hot = hotRef.current.hotInstance;

hot.setDataAtCell(0, 0, 'new value');
```

but only after `.current` is defined - which means the component must be already rendered.